### PR TITLE
August 2024 Variant Update

### PR DESCRIPTION
**Pull request by brain-frog**
## What are the changes?

August 2024 Variant Update

## Why am I doing these changes?

Supposed to

## What did change?

771 Files :)

## How to test the changes?

Play :)

## Checklist
- [x] **I'm using `beta` as my base branch**
- [x] There is no overlap with another PR?
- [x] The PR is self-contained and cannot be split into smaller PRs?
- [x] Have I provided a clear explanation of the changes?
- [x] Have I considered writing automated tests for the issue?
- [ ] Have I tested the changes (manually)?
    - [ ] Are all unit tests still passing? (`npm run test`)
- [x] Are the changes visual?
  - [ ] Have I provided screenshots/videos of the changes?

### CODE REVIEW

**Review by Tempo-anon — APPROVED** [review]
771 files changed LGTM

**Comment by CodeTappert** [comment]
Give me Arcanine Variants or i riot!

Not approved! /s

**Review by CodeTappert — APPROVED** [review]

**Review by torranx — APPROVED** [review]
108,494 lines changed === lgtm

**Review by flx-sta — APPROVED** [review]
images + jsons seem to work all good.
I'm just a bit confused that we end on an uneven number. Isn't it 1 image + 1 json?

File: public/images/pokemon/variant/back/618_2.png
Comment (Fontbane) [inline public/images/pokemon/variant/back/618_2.png:1]: This is wrong

**Review by Fontbane — CHANGES_REQUESTED** [review]

File: public/images/pokemon/variant/back/618_2.png
Comment (Fontbane) [reply public/images/pokemon/variant/back/618_2.png:1]: Ok, I looked into it, all the variants that have separate image files have base fixes, meaning those should just be regular palette swaps. The only outliers that should have a separate sprite for their variants are Alomomola and Gigalith. Something probably went wrong with the script that's supposed to generate these files. But if the variants work, they work.

**Pull request merged**